picard/util/pipe.py

```diff
@@ -392,42 +392,54 @@ def __init__(self, app_name: str, app_version: str, args: Optional[Iterable[str]
         except AttributeError:
             pass
         super().__init__(app_name, app_version, args, forced_path, identifier)
-
+        self.__create_pipe()
         self._remove_temp_attributes()
 
+    def __create_pipe(self):
+        try:
+            self.__pipe = win32pipe.CreateNamedPipe(
+                self.path,
+                win32pipe.PIPE_ACCESS_DUPLEX,
+                win32pipe.PIPE_TYPE_MESSAGE | win32pipe.PIPE_WAIT,
```

Comment:
See the note at the end of https://learn.microsoft.com/en-us/windows/win32/ipc/named-pipe-type-read-and-wait-modes

Reply:
Yes, that's correct. But the other reason why blocking mode is used here is just because it is the best way to handle it. With the blocking wait the thread waits until a client actually sends a message. That way the thread is just idle and does not consume CPU time until a message comes in.